forest_benchmarking/dfe.py

```diff
@@ -140,7 +192,7 @@ class DFEdata:
     count: List[int]
     """number of shots used to calculate the `expectation`"""
 
-def acquire_dfe_data(experiment: DFEexperiment, quantum_machine: QuantumComputer, var: float = 0.01) -> Tuple[DFEdata, DFEdata]:
+def acquire_dfe_data(experiment, quantum_machine: QuantumComputer, var: float = 0.01) -> Tuple[DFEdata, DFEdata]:
```

Comment:
Don't we need a shim for any of the additions to be used here (Why remove the DFEexperiment tag?)?